doc/code_style/mom.cpp

```diff
@@ -234,7 +223,8 @@ void YourMom::buy(const Thing & /*item*/) {
 	// TODO: implement
 }
 
-}} // parents::mom
+} // namespace mom
+} // namespace parents
```

Comment:
This is done automatically, so it's fine i think. Also it gives less errors later in `clang-tidy`